### PR TITLE
TM-5457, Panel Meeting Dates, use ET and be able to read without Zulu

### PR DESCRIPTION
**Pull request by M00staff**
Need to double check in dev - what sucks is that this will look off locally, since our local data uses the Zulu ... should work on dev, just need to double check

[Ticket](https://metaphase.atlassian.net/browse/TM-5457)

### CODE REVIEW

File: src/utilities.js [hunk 1]
```diff
@@ -322,7 +322,7 @@ export const getTimeDistanceInWords = (dateToCompare, date = new Date(), options
 // Format the date into our preferred format.
 // We can take any valid date and convert it into M.D.YYYY format, or any
 // format provided with the dateFormat param.
-export const formatDate = (date, dateFormat = 'MM/DD/YYYY') => {
+export const formatDate = (date, dateFormat = 'MM/dd/yyyy') => {
```
Comment (M00staff) [inline src/utilities.js:325]: touchup - this will actually throw an error with date-fns

File: src/Components/Panel/helpers.js [hunk 2]
```diff
@@ -77,9 +76,12 @@ export const submitPanelMeeting = (originalFields, newFields) => {
...
     panelMeetingType: newFields?.panelMeetingType,
-    panelMeetingDate: newFields?.panelMeetingDate,
-    prelimCutoff: newFields?.prelimCutoff,
-    addendumCutoff: newFields?.addendumCutoff,
+    panelMeetingDate: newFields?.panelMeetingDate
+      && format(newFields?.panelMeetingDate, 'MM/dd/yyyy HH:mm'),
+    prelimCutoff: newFields?.prelimCutoff
+      && format(newFields?.prelimCutoff, 'MM/dd/yyyy HH:mm'),
+    addendumCutoff: newFields?.addendumCutoff
+      && format(newFields?.addendumCutoff, 'MM/dd/yyyy HH:mm'),
```
Comment (M00staff) [inline src/Components/Panel/helpers.js:84]: dates will be passed to the server, like so - hope that doesn't bonk the Zulu stripping code 🤷

image

File: src/Components/Panel/helpers.js [hunk 3]
```diff
@@ -77,9 +76,12 @@ export const submitPanelMeeting = (originalFields, newFields) => {
...
     panelMeetingType: newFields?.panelMeetingType,
-    panelMeetingDate: newFields?.panelMeetingDate,
-    prelimCutoff: newFields?.prelimCutoff,
-    addendumCutoff: newFields?.addendumCutoff,
+    panelMeetingDate: newFields?.panelMeetingDate
+      && format(newFields?.panelMeetingDate, 'yyyy-MM-dd HH:mm:ss'),
+    prelimCutoff: newFields?.prelimCutoff
+      && format(newFields?.prelimCutoff, 'yyyy-MM-dd HH:mm:ss'),
+    addendumCutoff: newFields?.addendumCutoff
+      && format(newFields?.addendumCutoff, 'yyyy-MM-dd HH:mm:ss'),
```
Comment (M00staff) [inline src/Components/Panel/helpers.js:84]: image

Comment (scott062) [reply src/Components/Panel/helpers.js:84]: nice

**Review by scott062 — APPROVED** [review]

**Pull request merged**